server/migrations/4-fix-ttl.ts

```diff
@@ -0,0 +1,35 @@
+import { DataMapper } from '@aws/dynamodb-data-mapper';
+import { DynamoDB } from 'aws-sdk';
+import { getUnixTime } from 'date-fns';
+import { Config } from '../app-config';
+import { BookingsModel } from '../db/bookings';
+import { OfficeBookingModel } from '../db/officeBookings';
+import { UserBookingsModel } from '../db/userBookings';
+
+// Check if TTL is beyond the year 3000
+const timestampTooHigh = (ttl: number) => ttl > 32503680000;
```

Comment:
Just to confirm - is there any chance that this number may not be reasonable (i.e. there are timestamps less than this but still greater than the retention)